Dockerfile

```
@@ -1,4 +1,22 @@
-FROM python:3.13-slim-bookworm@sha256:21e39cf1815802d4c6f89a0d3a166cc67ce58f95b6d1639e68a394c99310d2e5
+
+# Build frontend
+FROM node:22-slim@sha256:bac8ff0b5302b06924a5e288fb4ceecef9c8bb0bb92515985d2efdc3a2447052 AS frontend-builder
+
+WORKDIR /frontend
+
+RUN corepack enable
+RUN corepack prepare pnpm@10.8.1 --activate
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)